project/BuildDependencies/scripts/0_package.target-win10-arm.list

```diff
@@ -6,6 +6,7 @@
 ; -> sqlite-3.7.12.1-win32\system\sqlite3.dll
 ; -> ...
 ;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
+cppwinrt-10.0.16299.0.7z
```

Comment:
If we require latest sdk that ships with VS2017 15.7 for building we don't have to ship this as it's included now.

Reply:
It's a plan. but currently I want to stay on 16299 and get feedback from alpha testers.

Reply:
Sounds smart